aspnetcore/signalr/authn-and-authz.md

```diff
@@ -19,6 +19,32 @@ By [Andrew Stanton-Nurse](https://twitter.com/anurse)
 
 SignalR can be used with [ASP.NET Core authentication](xref:security/authentication/identity) to associate a user with each connection. In a hub, authentication data can be accessed from the [`HubConnectionContext.User`](/dotnet/api/microsoft.aspnetcore.signalr.hubconnectioncontext.user) property. Authentication allows the hub to call methods on all connections associated with a user (See [Manage users and groups in SignalR](xref:signalr/groups) for more information). Multiple connections may be associated with a single user.
 
+The following is an example of `Startup.Configure` which uses SignalR and ASP.NET Core authentication:
+
+```csharp
+public void Configure(IApplicationBuilder app)
+{
+    ...
+
+    app.UseStaticFiles();
+    
+    app.UseAuthentication();
+
+    app.UseSignalR(hubs =>
```

Comment:
I'd like us to start planning for, in the 3.0 wave, to show how to route the SignalR hub endpoints using endpoint routing. This provides a more consistent API and a one-stop place for folks to wire up all the endpoints.